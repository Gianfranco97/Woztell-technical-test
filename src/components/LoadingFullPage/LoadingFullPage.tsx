import { Spin } from 'antd';
import MainLayout from 'components/MainLayout';
import "./LoadingFullPage.scss"

export default function LoadingFullPage() {
  return (
    <MainLayout>
      <div className='loading-container'>
        <Spin size="large" />
      </div>
    </MainLayout>
  )
}