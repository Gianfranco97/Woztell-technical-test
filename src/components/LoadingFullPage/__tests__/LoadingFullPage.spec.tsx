import { expect } from '@jest/globals';
import { render } from '@testing-library/react'
import LoadingFullPage from '../index';

it('The snapshot matches', () => {
  const { container } = render(
    <LoadingFullPage />
  )
  expect(container).toMatchSnapshot()
});
