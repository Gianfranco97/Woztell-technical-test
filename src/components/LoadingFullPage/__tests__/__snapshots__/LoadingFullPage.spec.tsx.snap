// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The snapshot matches 1`] = `
<div>
  <section
    class="ant-layout layout"
  >
    <header
      class="ant-layout-header layout__header"
    >
      <h1>
        Technical test - Woztell
      </h1>
    </header>
    <section
      class="ant-layout"
    >
      <main
        class="ant-layout-content layout__main"
      >
        <div
          class="loading-container"
        >
          <div
            aria-busy="true"
            aria-live="polite"
            class="ant-spin ant-spin-lg ant-spin-spinning"
          >
            <span
              class="ant-spin-dot ant-spin-dot-spin"
            >
              <i
                class="ant-spin-dot-item"
              />
              <i
                class="ant-spin-dot-item"
              />
              <i
                class="ant-spin-dot-item"
              />
              <i
                class="ant-spin-dot-item"
              />
            </span>
          </div>
        </div>
      </main>
    </section>
    <footer
      class="ant-layout-footer layout__footer"
    >
      Â© 2022 - Gianfranco Manganiello
    </footer>
  </section>
</div>
`;
